src/components/book/Book.js

```diff
@@ -1,13 +1,25 @@
+import { Link } from 'react-router-dom';
 import Author from '../author/Author';
+import BookDetail from './BookDetail.js';
 import './Book.css';
 
 function Book({
-  book: { title, total_pages: totalPages, rating, isbn, published_date: publishedDate, authors },
+  book: {
+    title,
+    total_pages: totalPages,
+    rating,
+    isbn,
+    published_date: publishedDate,
+    authors,
+    book_id,
+  },
   showDetail = false,
 }) {
   return (
     <article className="book">
-      <h3>{title}</h3>
+      <h3>
+        <Link to={`/books/${book_id}`}>{title}</Link>
```

Comment:
You don't want this to be the link because then you have a link on the detail page to the same page.